tests/test_serializers.py

```diff
@@ -116,11 +131,10 @@ def setUp(self, save_mock, manager_mock, get_queryset_mock):
         # This emulates property in the database, which needs to be updated
         # as a result of given test.
         self.existing_prop_mock = Mock(name='name', value='value')
-
-        filter_mock = Mock()
         manager_mock.filter = Mock(return_value=[self.existing_prop_mock])
 
         get_queryset_mock.return_value = Mock(get=Mock(return_value=1))
+        get_queryset_mock.__func__ = get_queryset_mock
```

Comment:
Seems like we're feeling pain because we mocked things owned by the framework: https://github.com/yola/drf-madprops/blob/f04f336a62028c384942d45e4cc64862d4fb6d58/tests/test_serializers.py#L127

That's a smell. I'd prefer to test serializers using real model instances. Would that get rid of the need to mock these things?